src/test/fixtures/fixtures_test.ts

```diff
@@ -123,27 +123,27 @@ suite('Fixtures', () => {
         assert.isOk(fs.statSync(fixtureSourceDir).isDirectory());
         assert.isOk(fs.statSync(fixtureExpectedDir).isDirectory());
         await fs.emptyDir(fixtureResultDir);
-        await fs.copy(fixtureSourceDir, fixtureResultDir, {
-          filter: (src) =>
-              !src.includes('bower_components') && !src.includes('node_modules')
-        });
+        await fs.copy(fixtureSourceDir, fixtureResultDir);
 
         // Top-Level Integration Test! Test the CLI interface directly.
-        const output = await exec(fixtureSourceDir, 'node', [
+        const output = await exec(fixtureResultDir, 'node', [
           modulizerBinPath,
           '--out',
-          '../generated/',
+          fixtureResultDir,
```

Comment:
can you do `'.'` here instead? That would be more true to how our users use it, and less likely to get out of sync with the first argument.

Reply:
Done.